.circleci/config.yml

```diff
@@ -15,33 +15,85 @@ orbs:
   slack: circleci/slack@3.4.2
 
 parameters:
+  # Configurable Versions
```

Comment:
Refactored - grouped some parameters together for better readability. The only new parameter is 'pre-publish'.